kong/plugins/http-log/sender.lua

```diff
@@ -75,6 +75,7 @@ local function send(self, bodies)
     })
     if not res then
       log(ERR, "failed request to ", host, ":", tostring(port), ": ", err)
+      return false
```

Comment:
Before exiting from this function, we still need the call to `set_keepalive()`.

Something like this may work:
```lua
local res, err = httpc:request(...)
if not res then
  log(ERR, ...)

else
  res:read_body()
end

ok, err = httpc:set_keepalive(...)
if not ok then
  log(ERR, ...)
end

return res ~= nil
```

Reply:
I dig this, will commit this version of it.

Reply:
I really should have expanded the diff here :joy: